packages/fx-core/src/common/tools.ts

```diff
@@ -417,7 +417,7 @@ export function isV3Enabled(): boolean {
 }
 
 export function isDownloadDirectoryEnabled(): boolean {
-  return process.env.DOWNLOAD_DIRECTORY === "true";
+  return process.env.DOWNLOAD_DIRECTORY ? process.env.DOWNLOAD_DIRECTORY === "true" : true;
```

Comment:
Since we want to always turn on this feature flag to monitor and collect data in the next release. Why don''t just remove this feature flag?

In this code, it still can go to download from big zip file if the env var is not true.

Reply:
Can you remove this switch or add another one?  
If you want to keep this switch, consider adding a new controller that can be configured to revert back to the original download method. This will make end-users debug more easily.

Reply:
I think setting it default to be true is ok. Since user won't know about this feature flag, it's almost impossible for them to turn it off by coincidence. Also we have the flexbility to revert it back to old method.